src/grandorgue/GOSetter.cpp

```diff
@@ -723,12 +763,28 @@ void GOSetter::Crescendo(int newpos, bool force) {
   if (pos == m_crescendopos)
     return;
 
+  bool crescendoAddMode = !m_CrescendoOverrideMode[m_crescendobank];
+
   while (pos > m_crescendopos) {
-    m_crescendo[++m_crescendopos + m_crescendobank * CRESCENDO_STEPS]->Push();
+    const unsigned oldIdx = m_crescendopos + m_crescendobank * CRESCENDO_STEPS;
```

Comment:
@oleg68 Wouldn't it be more optimized to have one while() call per mode (AddMode or Override) to avoid testing crescendoAddMode at each iteration ?

Reply:
> @oleg68 Wouldn't it be more optimized to have one while() call per mode (AddMode or Override) to avoid testing crescendoAddMode at each iteration ?

Might be, but I prefer to reuse common pieces of code as more as possible. This code is more understandable.

Reply:
Ok, we'll see with usage.